### PR TITLE
build scip on travis and test

### PR DESCRIPTION
**Pull request by mlubin**

### CODE REVIEW

**Comment by mlubin** [comment]
Voilà, la coche verte

File: test/REQUIRE [hunk 1]
```diff
@@ -0,0 +1,2 @@
+JuMP
```
Comment (leethargo) [inline test/REQUIRE]: Oh nice, I didn't know you could have a separate REQUIRE for tests in Julia packages.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 # SCIP.jl
 Julia interface to [SCIP](http://scip.zib.de) solver.
 
+[![Build Status](https://travis-ci.org/leethargo/SCIP.jl.svg?branch=ml%2Ftravis)](https://travis-ci.org/leethargo/SCIP.jl)
```
Comment (mlubin) [inline README.md]: probably want to point to master branch here?

Comment (leethargo) [inline README.md]: Sure, I just copy&pasted the link blindly from Travis :see_no_evil:

**Pull request merged**